sqlmesh/schedulers/airflow/operators/hwm_sensor.py

```diff
@@ -35,7 +35,10 @@ def __init__(
     def poke(self, context: Context) -> bool:
         dag_run = context["dag_run"]
 
-        target_snapshot = self._get_target_snapshot()
+        with util.scoped_state_sync() as state_sync:
+            target_snapshot = state_sync.get_snapshots([self.target_snapshot_info], refresh=True)[
```

Comment:
I think `refresh=True` is meaningless here since `util.scoped_state_sync()` always returns a new state sync instance, so there is no cache anyway.